PWGHF/HFC/TableProducer/correlatorD0Hadrons.cxx

```diff
@@ -146,24 +139,23 @@ struct HfCorrelatorD0HadronsSelection {
                                aod::McParticles const& mcParticles)
   {
     bool isD0Found = 0;
-    for (const auto& particle1 : mcParticles) {
-      if (std::abs(particle1.pdgCode()) != Pdg::kD0) {
-        continue;
-      }
-      double yD = RecoDecay::y(particle1.pVector(), MassD0);
-      if (yCandMax >= 0. && std::abs(yD) > yCandMax) {
+    for (const auto& particle : mcParticles) {
+      if (std::abs(particle.pdgCode()) != Pdg::kD0) {
```

Comment:
Is it foreseen that you don't check the decay channel here?

Reply:
Yes, we are aware of that. Actually, the MCgen part is yet to be updated and tested fully. Another PR will be created once done.